src/app/js/formats/shared/LodexResource.js

```diff
@@ -35,22 +35,46 @@ const styles = StyleSheet.create({
 });
 
 // see https://jsonfeed.org/version/1#items
-const LodexResource = ({ id, title, summary }) => {
+const LodexResource = ({ id, url, title, summary }) => {
     if (!id) {
         return null;
     }
-    return (
-        <div id={id}>
-            <Link
-                className={css(styles.contentLink)}
-                to={getFullResourceUri({ uri: id })}
-            >
+    let urlDomain = url.replace(id, '');
+    let getFullResourceUriDomain = getFullResourceUri({ uri: id }).replace(
+        id,
+        '',
+    );
+
+    const content = () => {
```

Comment:
Why make `content` a function?
It could be a variable containing a JSX structure. I think it would work as well.